test/window_test.cpp

```diff
@@ -95,8 +24,8 @@ bool check_symmetry()
     int begin = random_integer(0, size / 2 + 2);
     int end = random_integer(size / 2 - 2, size);
     
-    double window1[size];
-    double window2[size];
+    double* window1 = (double*) malloc(sizeof(double) * size);
```

Comment:
std::vector would probably be better here (just use .data() if you need the raw pointer)

Reply:
If it relates to FFT stuff it should actually call htl::allocate_aligned() and html::deallocate_aligned(). Otherwise std:vector<> would be OK.